doc/visual-programming/source/loading-your-data/index.rst

```diff
@@ -133,45 +118,41 @@ This is how the header with augmented attribute names looks like in
 Excel (:download:`sample-head.xlsx <sample-head.xlsx>`):
 
 .. image:: spreadsheet-simple-head1.png
-    :width: 500 px
+    :width: 414 px
     :align: center
 
-We can again use a `Data Table <http://docs.orange.biolab.si/widgets/rst/data/datatable.html#data-table>`_ widget to read the data from Excel file. Orange will automatically recognize attribute values, which is evident in the modified class icons:
+We can again use again a **File** widget to load this data set and then render it in the **Data Table**:
```

Comment:
Actually data set can be written as 2 words or 1:
https://en.wikipedia.org/wiki/Data_set

2 words is probably slightly more "proper" for now. It is common in English for phrases to start as 2 words and then through frequent usage they merge into a single word (often there is an intermediate version with a hyphen).

Anyway the most important thing is to be consistent and always use the same choice.